pkgs/development/libraries/libtiff/default.nix

```diff
@@ -19,6 +20,19 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "dev" "out" "man" "doc" ];
 
+  patches = [
+    (fetchpatch {
```

Comment:
Both patches are merged so I think it should be alright to fetch a MR but individual commits are usually preferable.